server/src/main/java/org/elasticsearch/common/xcontent/XContent.java

```diff
@@ -106,7 +113,10 @@ XContentParser createParser(NamedXContentRegistry xContentRegistry,
 
     /**
      * Creates a parser over the provided bytes.
+     * @deprecated use {@link #createParser(NamedXContentRegistry, DeprecationHandler, InputStream)}
+     * instead, the BytesReference will be removed it a future commit
```

Comment:
s/it/in , also not sure what you mean with "the BytesReference  will be removed in a future commit", maybe the sentence needs to be clarified.